### DISCUSSION TITLE
New plugin: f5log

### DISCUSSION THREAD

**Discussion by bitwisecook** [saulpw/visidata#1258] [Show and tell]
I've hacked together a log parser for f5 logs, https://github.com/bitwisecook/visidata-f5log

What's the best way to enable this as an installable plugin?

Thanks!

**Comment by geekscrapy** [comment]
This looks great, can't wait to try it!
Best place to start is here: https://www.visidata.org/docs/api/plugins
The last bullet point on that page says about making a PR to add the plugin to the official visidata plugin repo if that's what you'd like 👌

**Comment by saulpw** [comment]
This is really awesome, @bitwisecook. It looks clean and well put-together.  I would love to see it in action!  I used to work at F5 but I don't have F5 logs myself anymore.  Any chance you could post a short video of what it's like to use it?

Also @geekscrapy is right, that's the current process.  I'm thinking we should update it a bit, and if you're interested in talking it through, your feedback would be much appreciated.  [We're on [Discord chat](https://visidata.org/chat) and libera.chat IRC among other places.]

**Comment by bitwisecook** [comment]
PR: https://github.com/saulpw/visidata/pull/1253

I've added basic colourisation to 0.2.0 but I don't yet understand how to add new colours (like a dark/light grey)
